### PR TITLE
feat(e2e-connector-test): replicate the react app in next FE-780

### PR DESCRIPTION
**Pull request by nelitow**
Goal:
Use the `react-app` components inside a Next instance and run the e2e on that with the same test suite as the React app.

### CODE REVIEW

**Comment by helciofranco** [comment]
e2e not passing anymore after our latest changes and there are those 2 details to fix

**Review by helciofranco — DISMISSED** [review]

File: e2e-tests/runner/.env.example [hunk 1]
```diff
@@ -1,4 +1,5 @@
-PORT=5173
+REACT_APP_PORT=
+REACT_NEXT_PORT=
 VITE_FUEL_PROVIDER_URL="https://testnet.fuel.network/v1/graphql"
```
Comment (arthurgeron) [inline e2e-tests/runner/.env.example:3]: Does it really use these env variables?

File: e2e-tests/react-next/.env.example [hunk 2]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_WC_PROJECT_ID=
```
Comment (arthurgeron) [inline e2e-tests/react-next/.env.example:1]: best to add `NEXT_PUBLIC_PROVIDER_URL=https://testnet.fuel.network/v1/graphql` here

**Review by arthurgeron — CHANGES_REQUESTED** [review]
Not sure if this is only on my side, but web3modal is not loading some logos:
CleanShot 2024-11-07 at 12 21 41@2x

**Comment by nelitow** [comment]
> Not sure if this is only on my side, but web3modal is not loading some logos: CleanShot 2024-11-07 at 12 21 41@2x

@arthurgeron can you let me know what is the error on your side? It loads for me.

**Comment by arthurgeron** [comment]
> > Not sure if this is only on my side, but web3modal is not loading some logos: CleanShot 2024-11-07 at 12 21 41@2x
> 
> @arthurgeron can you let me know what is the error on your side? It loads for me.

Seems like it failed with a CORS error:
CleanShot 2024-11-07 at 20 32 58@2x

**Review by arthurgeron — APPROVED** [review]

**Pull request merged**